config/config.go

```diff
@@ -115,3 +116,16 @@ type Cert struct {
 	CertFile string `envconfig:"CERT_FILE"`
 	KeyFile  string `envconfig:"KEY_FILE"`
 }
+
+type Metrics struct {
```

Comment:
__[golint]__
* [golint] exported type Metrics should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->